cluster/gce/gci/mounter/BUILD

```diff
@@ -0,0 +1,34 @@
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])
+
+load(
+    "@io_bazel_rules_go//go:def.bzl",
+    "go_binary",
+    "go_library",
+)
+
+go_binary(
+    name = "mounter",
```

Comment:
this is currently statically linked - is it still statically linked when built each time?